src/sentry/integrations/slack/message_builder/issues.py

```diff
@@ -225,6 +225,8 @@ def get_color(
             return "info"
     if event_for_tags:
         color: str | None = event_for_tags.get_tag("level")
+        if hasattr(event_for_tags, "occurrence") and event_for_tags.occurrence is not None:
```

Comment:
Probably also needs `and event_for_tags.occurrence.level is not None` since it's optional.